src/PanoImageRenderer/renderer/SphereRenderer.js

```diff
@@ -50,27 +43,48 @@ export default class SphereRenderer extends Renderer {
 			}`;
 	}
 
+	static _getPixelSource(image) {
+		if (!pixelCanvas) {
+			return image;
+		}
+		const ctx = pixelCanvas.getContext("2d");
```

Comment:
@happyhj I found meaningful performance difference. 
https://jsperf.com/cost-of-getcontext